include/zephyr/bluetooth/iso.h

```diff
@@ -217,6 +217,18 @@ struct bt_iso_recv_info {
 	uint8_t flags;
 };
 
+/** @brief ISO Meta Data structure for transmitted ISO packets. */
+struct bt_iso_tx_info {
```

Comment:
This is rather a comment on the already existing code:
I see this function uses "tx" in the name, and that seems to be used elsewhere in the file as well.
Should the previous struct be "_rx_info" instead of "recv" for consistency?

Reply:
This struct was originally named `bt_iso_transmit_info` to be somewhat in line with the `recv_info` naming, and then changed to `bt_iso_tx_info` based on PR feedback

Reply:
I do not have a strong opinion besides that I don't want to rename the `recv_info`, as `recv_info` is a common name given to the set of structs used in `recv` callbacks that provide additional information. 

We can keep this as `transmit`, but in that case I would prefer the function to also use `transmit`. Using `tx` and `transmit` in the same "case" is worse than using `tx/tx` or `transmit/transmit` IMO.

Reply:
Agree with Emil.